src/Library/demos/Avatar/main.js

```diff
@@ -0,0 +1,28 @@
+import Adw from "gi://Adw";
+import Gtk from "gi://Gtk";
+import Gdk from "gi://Gdk";
+
+const avatar_image = workbench.builder.get_object("avatar_image");
+const button = workbench.builder.get_object("button");
+
+const dialog = new Gtk.FileDialog();
+
+const filter = new Gtk.FileFilter();
+filter.name = "Image files";
```

Comment:
the filter name doesn't seem to work – this is what I see in the bottom right of the dialog

![image](https://user-images.githubusercontent.com/19673/222009456-a0f8420a-059d-4be0-b0a0-b072b20c60e4.png)

could you look into it?

try 

```
new Gtk.FileFilter({
  name: "something"
})
```

also "Images" is fine - no need to be overly specific

Reply:
```
new Gtk.FileFilter({
  name: "something"
})
```
This does not seem to solve the issue. I'll still just leave it in, if it gets fixed in the future.